magda-web-client/src/actions/regionMappingActions.js

```diff
@@ -38,11 +38,13 @@ export function fetchRegionMapping() {
       }
     })
     .then((json: FacetSearchJson) =>{
-      if(!json.error){
-        return dispatch(receiveRegionMapping(json));
-      } else{
-          return dispatch(requestRegionMappingError(json.error))
-      }
+        if(!json.error){
+          return dispatch(receiveRegionMapping(json));
+        } else{
+            return dispatch(requestRegionMappingError(json.error))
+        }
+      },(error)=>{
```

Comment:
Nitpicking - I think it's a bit clearer to use `.catch` for errors on promise chains rather than the second argument to `then`

Reply:
No problem. Will move the code to a seperate .catch.

Reply:
done in commit 2ea4414aa536606b9fa7630b2a2f83a4f95da850